examples/ads.amp.html

```diff
@@ -1580,6 +1581,13 @@ <h2>Rambler&Co</h2>
       json='{"params":{"p1":"bvpkq","p2":"y","pct":"a"}}'>
   </amp-ad>
 
+  <h2>Rambler&Co RNetPlus</h2>
+  <amp-ad width="300" height="400"
+      type="rnetplus"
+      layout="responsive"
+      src="https://api.rnet.plus/Scripts/rnet_amp_embed.js?blockId=660">
```

Comment:
It is up to you but you may want to hardcode this url, and have the block-id be a separate attribute? May be less likely for your users to break.

Reply:
Actually, it is little bit `corporate specific`, I am sorry. 

In few words, without RNetPlus team, you have no chances to place certain `block` to your pages with desired content; the empty blocks displays nothing. So, from the other side of this tube, somebody need to collect couple of teasers to display and assign to the particular block. 

As result, RNetPlus Team is always involved into embedding the teasers and it provides the code snippets to place teasers for AMP case and for casual web cases. 

It means the exact form is not important.

But if the form will be important for somebody I able to redesign this piece, thank you.

Reply:
SGTM.